### ISSUE TITLE
As a user, I want to be able to have both a General Revision and Amendment open at the same time, so that I can make general changes without being blocked by an in-progress amendment

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-cif#1580]
#### Description:

Currently, users can have only one revision open at a time. This card is about 
1. allowing them to have one of each type (General Revision and Amendment) open at the same time.
2. allowing only those two types

- Prototype: [New Revision](https://www.figma.com/proto/PzOY8RyQnVXnjZJdSv2AqQ/CIF---Internal-User-Portal?page-id=2289%3A2663&node-id=3169-2715&viewport=-470%2C1174%2C0.2&scaling=scale-down&starting-point-node-id=3169%3A2715&show-proto-sidebar=1)

#### Acceptance Criteria:

Given that I am a user
When I click the "New Revision" button
I have only two radio buttons, Amendment and General Revision

Given that I am a user
When I click the "New Revision" button and I have an open General Revision
Then I can create a new Amendment
and I cannot create a second General Revision
(the radio button for General Revision is greyed out with help text next to it "You have a General Revision currently in progress.")

Given that I am a user
When I have an open Amendment
Then when I click the "New Revision" button I can open a new General Revision
and I cannot open a second Amendment
(the radio button for Amendment is greyed out with help text next to it "You have an Amendment currently in progress.")

#### Development Checklist:

- [ ] If possible, remove the minor revision type from the db
- [ ] Remove the minor revision radio button from the form
- [ ] Edit the db constraint re no multiple revisions to allow one revision of each type
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

**Comment by nanyangpro** [comment]
Thanks for creating it, @BCerki - Your ???? are filled out with the link to the updated prototype (with Minor Revision removed) in the description.

**Comment by Dianadec** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1580?planning-poker) with Zenhub @BCerki @dleard @gurjmatharu @mikevespi @pbastia @sam-warren @Sepehr-Sobhani @tmastrom

**Comment by pbastia** [comment]
Double check with @mikevespi if the code already exists

**Comment by Dianadec** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1580?planning-poker) with Zenhub @JoshLarouche

**Comment by pbastia** [comment]
Thank you @mikevespi ! AC work  as intended
